protocol/streaming/full_node_streaming_manager.go

```diff
@@ -295,27 +311,28 @@ func (sm *FullNodeStreamingManagerImpl) SendOrderbookUpdates(
 	}
 
 	// Unmarshal each per-clob pair message to v1 updates.
-	updatesByClobPairId := make(map[uint32][]clobtypes.StreamUpdate)
+	updatesByClobPairId := make([]clobtypes.StreamUpdate, 0)
```

Comment:
rename var since this is not a map anymore?

Reply:
done